fieldsolver/ldz_volume.cpp

```diff
@@ -67,6 +67,95 @@ void calculateVolumeAveragedFields(
             volGrid0->at(fsgrids::volfields::PERBXVOL) = perturbedCoefficients[Rec::a_0];
             volGrid0->at(fsgrids::volfields::PERBYVOL) = perturbedCoefficients[Rec::b_0];
             volGrid0->at(fsgrids::volfields::PERBZVOL) = perturbedCoefficients[Rec::c_0];
+
+            // Calculate volume average of E (FIXME NEEDS IMPROVEMENT):
```

Comment:
Öh, what does this "FIXME NEEDS IMPROVEMENT" refer to?

Reply:
The vol E is computed as the average of the four edge values, not as the proper Balsara reconstruction.